.tools/validation/schema/example_schema.yaml

```diff
@@ -3,36 +3,58 @@
 
 map(include('example'), key=example_id())
 ---
+# An example blocks all the languages together for a single example in a tab list. It is a navigable page on the code examples library. It is the top level "unit" of SoS content. This metadata is merged from tributaries with aws-doc-sdk-examples.
 example:
-  title: str(upper_start=True, no_end_punc=True)
-  title_abbrev: str(upper_start=True, no_end_punc=True)
-  synopsis: str(required=False, lower_start=True, end_punc_or_semicolon=True)
-  synopsis_list: list(str(upper_start=True, end_punc=True), required=False)
-  category: str(required=False, upper_start=True, no_end_punc=True)
-  guide_topic: include('guide_topic', required=False)
+  # Human readable title, defaulting to slug-to-title of the ID if not provided. Overwritten by aws-doc-sdk-example when merging.
```

Comment:
I'm somewhat uneasy with these comments that are about our future feature intent (i.e., slug-to-title) rather than what actually exists today.

Reply:
Done. TODO'd things that IMHO tributaries need to keep in mind today, but we haven't implemented yet. Lemme know if that's enough, or if you want me to remove them entirely.

Reply:
Done. Marked things not yet implemented that I think tributaries should be aware of as TODO, but I'd remove them if you preferred.